### PR TITLE
MC Upload task should support UWP app symbols

### PR DESCRIPTION
**Pull request by AlexRukhlin**
uploadtask

### CODE REVIEW

File: Tasks/VSMobileCenterUpload/task.json [hunk 1]
```diff
@@ -68,7 +68,17 @@
...
+            "visibleRule": "symbolsType == AndroidNative"
+        },
+        {
+            "name": "pdbPath",
+            "type": "filePath",
+            "label": "PDB Files Path Pattern",
+            "defaultValue": "**/*.pdb",
+            "groupName": "symbols",
+            "required": false,
+            "helpMarkDown": "Relative path pattern from the repo root to PDB symbols files.",
```
Comment (madhurig) [inline Tasks/VSMobileCenterUpload/task.json]: It need not be a pattern. So I think it is better to say "Relative path from the repo root to the PDB symbols files. Path may contain wildcards".

File: Tasks/VSMobileCenterUpload/task.json [hunk 2]
```diff
@@ -68,7 +68,17 @@
...
+            "visibleRule": "symbolsType = Windows"
```
Comment (madhurig) [inline Tasks/VSMobileCenterUpload/task.json]: Should we rename Windows to UWP or use Windows/UWP in the strings visible to the user?

File: Tasks/VSMobileCenterUpload/task.json (hunk 1 above)
Comment (DavidStaheli) [inline Tasks/VSMobileCenterUpload/task.json]: Building on Madhuri's suggestion:
Enter the relative path from the root of the repository to the .pdb symbols files. You may include wildcards, such as "**/*.pdb".

File: Tasks/VSMobileCenterUpload/task.json (hunk 2 above)
Comment (DavidStaheli) [inline Tasks/VSMobileCenterUpload/task.json]: I suggest changing the non-visible dropdown value to "UWP" and changing the visible text to "Universal Windows Platform (UWP)".

File: Tasks/VSMobileCenterUpload/task.json [hunk 3]
```diff
@@ -68,7 +68,17 @@
             "groupName": "symbols",
             "required": false,
             "helpMarkDown": "Relative path from the repo root to the symbols folder.",
-            "visibleRule": "symbolsType = Windows || symbolsType == AndroidNative"
+            "visibleRule": "symbolsType == AndroidNative"
+        },
+        {
+            "name": "pdbPath",
+            "type": "filePath",
+            "label": "PDB Files Path Pattern",
```
Comment (DavidStaheli) [inline Tasks/VSMobileCenterUpload/task.json]: Suggestion for this label:
"Symbols Path (*.pdb)"
Then users have a hint, in addition to the helpMarkDown, about what PDB files are.

**Pull request merged**